packages/clerk-js/src/ui/components/OrganizationProfile/VerifiedDomainPage.tsx

```diff
@@ -207,22 +297,36 @@ export const VerifiedDomainPage = withCardStateProvider(() => {
             {allowsEdit && (
               <TabPanel
                 direction={'col'}
-                sx={[
-                  { width: '100%' },
-                  t => ({
-                    padding: `${t.space.$none} ${t.space.$4}`,
-                  }),
-                ]}
+                gap={4}
+                sx={{ width: '100%' }}
               >
-                <LinkButtonWithDescription
-                  title={localizationKeys('organizationProfile.verifiedDomainPage.dangerTab.removeDomainTitle')}
-                  subtitle={localizationKeys('organizationProfile.verifiedDomainPage.dangerTab.removeDomainSubtitle')}
-                  actionLabel={localizationKeys(
-                    'organizationProfile.verifiedDomainPage.dangerTab.removeDomainActionLabel__remove',
-                  )}
-                  colorScheme='danger'
-                  onClick={() => navigate(`../../domain/${domain.id}/remove`)}
-                />
+                {dangerCalloutLabel && (
+                  <CalloutWithAction
+                    text={dangerCalloutLabel}
+                    icon={
+                      <Icon
+                        colorScheme='neutral'
+                        icon={InformationCircle}
+                        sx={t => ({ marginTop: t.space.$1 })}
+                      />
+                    }
+                  />
```

Comment:
Not needed to do this in the context of this PR, but as a future improvement, the CalloutWithAction component could accept a component ctor as a prop, eg `icon={InformationCircle}` and handle the styling internally